contracts/src/Types.sol

```diff
@@ -107,4 +111,5 @@ struct Ticket {
 
 struct TokenInfo {
     bool isRegistered;
```

Comment:
To ensure storage is not corrupted after upgrades, we need to preserve the storage layou, only adding new fields at the end.

```suggestion
    bool isRegistered;
    bytes31 __padding;
```